packages/manager-components/src/components/table/table.spec.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { within } from '@testing-library/react';
+import { OsdsText } from '@ovhcloud/ods-components/text/react';
+import { ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { Table, TableProps } from './table.component';
+import { mockColumns, mocksData } from './table.mock';
+import { render } from '../../utils/test.provider';
+
+const setupSpecTest = async (
+  customProps?: Partial<TableProps>,
+  table?: TableProps,
+) =>
+  render(
+    <Table {...customProps}>
+      <thead>
```

Comment:
just a comment regarding this test :

since your component is just wrapping <table> and allow any type of {children} inside it's not necessary to make this kind of unit test

you basically just have to check if the children passed in props is the same as the children rendered ; there are 0 logic regarding rows in the component

no need to refactor the test since it's not a blocker for the PR but i just wanted to make it clear that it's kind of a misleading unit test since most of the logic tested is not logic inside your table component